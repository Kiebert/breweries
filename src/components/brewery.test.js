import React from 'react';
import { shallow } from 'enzyme';
import Brewery from './brewery';

describe('<Brewery />', () => {
    it('renders brewery information', () => {

        const json = {
            name: "Brouwerij Dampegheest",
            address: "Achterweg 22",
            zipcode: "1906 AG",
            city: "Limmen",
            open: ["Friday", "Saturday", "Sunday"]
        }

        const component = shallow(
            <Brewery data={json} />
        );

        expect(component).toMatchSnapshot();


    });

});