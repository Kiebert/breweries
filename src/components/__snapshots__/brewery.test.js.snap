// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Brewery /> renders brewery information 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="Popup"
    onClick={undefined}
  >
    <div
      className="Brewery"
    >
      <div
        className="Brewery-Name"
      >
        Brouwerij Dampegheest
         
      </div>
      <div>
        Address: 
        Achterweg 22
         
      </div>
      <div>
        Zipcode: 
        1906 AG
         
      </div>
      <div>
        City: 
        Limmen
         
      </div>
      <div>
        Friday
      </div>
      <div>
        Saturday
      </div>
      <div>
        Sunday
      </div>
      <div
        className="btnClose"
      >
        X
      </div>
    </div>
  </div>,
  "nodes": Array [
    <div
      className="Popup"
      onClick={undefined}
    >
      <div
        className="Brewery"
      >
        <div
          className="Brewery-Name"
        >
          Brouwerij Dampegheest
           
        </div>
        <div>
          Address: 
          Achterweg 22
           
        </div>
        <div>
          Zipcode: 
          1906 AG
           
        </div>
        <div>
          City: 
          Limmen
           
        </div>
        <div>
          Friday
        </div>
        <div>
          Saturday
        </div>
        <div>
          Sunday
        </div>
        <div
          className="btnClose"
        >
          X
        </div>
      </div>
    </div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <Brewery
        data={
          Object {
            "address": "Achterweg 22",
            "city": "Limmen",
            "name": "Brouwerij Dampegheest",
            "open": Array [
              "Friday",
              "Saturday",
              "Sunday",
            ],
            "zipcode": "1906 AG",
          }
        }
      />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": Brewery {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {
          "data": Object {
            "address": "Achterweg 22",
            "city": "Limmen",
            "name": "Brouwerij Dampegheest",
            "open": Array [
              "Friday",
              "Saturday",
              "Sunday",
            ],
            "zipcode": "1906 AG",
          },
        },
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="Popup"
          onClick={undefined}
        >
          <div
            className="Brewery"
          >
            <div
              className="Brewery-Name"
            >
              Brouwerij Dampegheest
               
            </div>
            <div>
              Address: 
              Achterweg 22
               
            </div>
            <div>
              Zipcode: 
              1906 AG
               
            </div>
            <div>
              City: 
              Limmen
               
            </div>
            <div>
              Friday
            </div>
            <div>
              Saturday
            </div>
            <div>
              Sunday
            </div>
            <div
              className="btnClose"
            >
              X
            </div>
          </div>
        </div>,
        "_debugID": 2,
        "_renderedOutput": <div
          className="Popup"
          onClick={undefined}
        >
          <div
            className="Brewery"
          >
            <div
              className="Brewery-Name"
            >
              Brouwerij Dampegheest
               
            </div>
            <div>
              Address: 
              Achterweg 22
               
            </div>
            <div>
              Zipcode: 
              1906 AG
               
            </div>
            <div>
              City: 
              Limmen
               
            </div>
            <div>
              Friday
            </div>
            <div>
              Saturday
            </div>
            <div>
              Sunday
            </div>
            <div
              className="btnClose"
            >
              X
            </div>
          </div>
        </div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Brewery
    data={
      Object {
        "address": "Achterweg 22",
        "city": "Limmen",
        "name": "Brouwerij Dampegheest",
        "open": Array [
          "Friday",
          "Saturday",
          "Sunday",
        ],
        "zipcode": "1906 AG",
      }
    }
  />,
}
`;
